src/features/import/utils/remapFields.ts

```diff
@@ -0,0 +1,24 @@
+import { ImportPreview, PersonImport } from './types';
+
+type TransformFunction = (k: string) => string;
+
+const transform: TransformFunction = (k) => (k === 'member_id' ? 'ext_id' : k);
+
+const objectRecursiveKeyMap = (obj: object, fn: TransformFunction) =>
+  Object.fromEntries(
+    Object.entries(obj).map(([key, value]) => {
+      const getValue = (v: unknown): unknown =>
+        typeof v === 'object' && v !== null ? objectRecursiveKeyMap(v, fn) : v;
+      return [
+        fn(key),
+        Array.isArray(value)
+          ? value.map((val) => getValue(val))
+          : getValue(value),
+      ];
+    })
+  );
+
+const remapFields = (importRes: ImportPreview | PersonImport) =>
```

Comment:
It took some effort understanding what this function does, but in the end I understand that it translates the key `member_id` (which is incorrectly leaked from the backend) to `ext_id` which is what it should be called.